pages/student/edit.vue

```diff
@@ -0,0 +1,325 @@
+            </v-col>            
+          </v-form>
+          <v-snackbar
+            v-model="snackbar"
+            :color="snackbarStatus"
+            :timeout="5000"
+            :top="true"
+            :right="true"
+          >
+            {{ snackbarText }}
+            <v-btn
+              color="#FFF"
+              text
+              @click="snackbar = false"
+            >
+              Fechar
+            </v-btn>
+          </v-snackbar>
+        </v-row>
+      </v-container>
+    </v-flex>
+  </v-layout>
+</template>
+
+<router>
+  {
+    path: '/aluno/alterar'
+  }
+</router>
+
+<script scoped>
+import auth from "../../services/http/auth";
+import  users  from '../../services/http/users';
+
+
+export default {
+  data() {
+    return {
+      status: true,
+      loading: false,            
+      snackbar: false,
+      snackbarText: '',
+      snackbarStatus: '',
+      token: '',
+      form: {
+        id: '',
+        name: '',
+        email: '',
+        urlFacebook: '',
+        urlInstagram: '',
+        role: ''
+      },
+      nameRules: [v => !!v || "Digite seu nome"],
+      emailRules: [
+        v => !!v || "Digite o e-mail",
+        v => /.+@.+\..+/.test(v) || "E-mail inválido"
+      ]
+    };
+  },
+  created(){       
+    let user = this.getUser();
+    this.form = { 
+      ...user,
+      role: 'STUDENT'
+    };
+  },
+   methods: {
+    submit() {
+      if (this.$refs.form.validate()) {
+        this.animateForm(true);
+          users.update(this.form)
+          .then(res => {     
+            this.loading = false;  
+            this.confirmSnackbar('Dados alterados com sucesso', 'success');
+            setTimeout(() => {
+              this.gotoIndex();
+            }, 1500); 
+          })
+          .catch(err => {
+            this.confirmSnackbar('Ocorreu um erro.', 'error');
+            setTimeout(() => {
+              this.loading = false;
+            }, 500);
+            console.error(err);
+          });
+      } else {
+        this.animateForm(false);
+      }
+    },
+
+    animateForm(status) {
+      if (status) {
+        this.$refs.flex.classList.add("hide-form");
+        document.querySelector("html").style.overflow = "hidden";
+        setTimeout(() => {
+          this.loading = true;
+        }, 300);
+      } else {
+        this.$refs.flex.classList.add("error-form");
+        setTimeout(() => {
+          this.$refs.flex.classList.remove("error-form");
+        }, 500);
+      }
+      document.querySelector("html").style.overflow = "scroll";
+    },        
+
+    gotoIndex() {
+      $nuxt._router.push("/index");
+    },
+
+    confirmSnackbar(text, status) {
+      this.snackbarText = text;
+      this.snackbarStatus = status;
+      this.snackbar = true;
+    },
+
+    getUser() {
+      try {
+        return JSON.parse(localStorage.getItem('user'))
+        
+      } catch (e) {
+        return {
+          name: '',
+          email: ''
```

Comment:
```suggestion
          email: '',
          urlFacebook: '',
          urlInstagram: ''
```
Completando os dados caso não consiga pegar do localStorage